Sources/GraphQLLanguage/Visitor.swift

```diff
@@ -41,24 +41,24 @@ extension Visitable {
             // These are known child field representation in each `Visitable`
             switch childValue {
             case let visitable as Visitable:
-                visitable.visit(with: visitor)
+                try visitable.visit(with: visitor)
             case let array as [Visitable]:
                 for visitable in array {
-                    visitable.visit(with: visitor)
+                    try visitable.visit(with: visitor)
                 }
             // See `ObjectValue`
             case let dictionary as [String: Visitable]:
                 for (_, visitable) in dictionary {
-                    visitable.visit(with: visitor)
+                    try visitable.visit(with: visitor)
                 }
             default:
                 break
             }
         }
-        visitor.visit(on: self)
+        try visitor.visit(on: self)
     }
 
-    public func visit(with visitor: @escaping (Visitable) -> Void) {
-        visit(with: BlockVisitor(block: visitor))
+    public func visit(with visitor: @escaping (Visitable) -> Void) throws {
```

Comment:
Probably this API want to have `throws` as well for the `@escaping (Visitable) -> Void`?

Reply:
Sounds like a good idea for future proofing. Will update.